app/views/works/_title_and_dates.html.erb

```diff
@@ -0,0 +1,49 @@
+<header class="show-title">
```

Comment:
we talked about making this a ViewModel so it has it's own helpers that go with it (and they don't need to be in the decorator), rather than just a template. And thus make the decorator(s) have to have a lot less stuff. What do you think?

Reply:
Done. It only really removes one method from the decorator, but it's somewhat better.